docs/quickstart.md

```diff
@@ -32,47 +32,28 @@ These instructions help you install the CLI.
 
 ### Install `conda` and `conda-forge`
 
-The easiest way to install MyST is with [the `conda` package manager](https://conda.io/projects/conda/en/latest/).
-`conda` is a multi-language manager that is useful for users across many data science languages like Python, R, Julia, and JavaScript.
+The easiest way to install MyST is with [the `mamba`/`conda` package manager](https://mamba.readthedocs.io/en/latest/index.html).
```

Comment:
```suggestion
The easiest way to install MyST is with [the `mamba`/`conda` package manager](https://mamba.readthedocs.io/en/latest/index.html).
```

I'd recommend just using mamba, or conda, but not both. We should assume that a user has no background in any of the installation methods we're talking about. Introducing multiple tools or concepts will just increase cognitive load. I'd suggest we just pick one way that is reasonable and explain that here, and then defer to `installing.md` for all the options, explanation, etc.

Reply:
Also just noting the section calls it `conda` but the instructions call it `mamba` . This will confuse users who do not understand the relationship between the two projects (this is 99% of users).